client/src/components/client-page/index.jsx

```diff
@@ -0,0 +1,846 @@
+    setEditedStatus12(event.target.value);
+  };
+
+
+  const handleChipDelete = (chipType) => {
+    switch (chipType) {
+      case 'status':
+        setEditedStatus('');
+        break;
+      case 'statusExit':
+        setEditedStatusExit('');
+        break;
+      case 'status3':
+        setEditedStatus3('');
+        break;
+      case 'status6':
+        setEditedStatus6('');
+        break;
+      case 'status9':
+        setEditedStatus9('');
+        break;
+      case 'status12':
+        setEditedStatus12('');
+        break;
+      default:
+        break;
+    }
+  };
+
+  const handleSave = () => {
+    
+    const updatedClientInfo = {
+      ...clientInfo,
+      firstName: editedName,
+      email: editedEmail,
+      phone: editedPhone,
+      status: editedStatus,
+      status_at_exit: editedStatusExit,
+      status_at_3: editedStatus3,
+      status_at_6: editedStatus6,
+      status_at_9: editedStatus9,
+      status_at_12: editedStatus12,
+    };
+
+    
+    onSaveChanges(updatedClientInfo);
+    setIsEditMode(!isEditMode);
+  };
+
+  const isStatusExitVisible = editedStatus === 'Closed';
+
+  return (
+    <div style={{ marginLeft: "40px", marginRight: "40px", marginTop: "40px" }}>
+      <div
+        style={{ display: "flex", alignItems: "center", marginBottom: "10px" }}
+      >
+        <IconButton onClick={handleBackClick} 
+        sx={{ color: "gray", marginRight: 2, marginLeft: 2 }}>
+        <ArrowBackIcon/>
+        </IconButton>
+        <Box className="title">
+          <Typography marginBottom={1} variant="h4" align="left">
+            {clientInfo.firstName}
+          </Typography>
+          <Typography variant="subtitle2" align="left">
+            Client
+          </Typography>
+        </Box>
+              <Box
+            sx={{
+              marginLeft: "auto",
+              display: "flex",
+              alignItems: "center",
+              borderRadius: "8px",
+              boxShadow: 2,
+              p: 3,
+            }}
+          >
+            <Box sx={{ textAlign: "center", mr: 2 }}>
+              <Typography variant="subtitle2" sx={{ mb: 1 }}>
+                Owner
+              </Typography>
+              <UserChipComponent user={owner} onClick={handleClick} edit />
+            </Box>
+            <Box sx={{ textAlign: "center" }}>
+              <Typography variant="subtitle2" sx={{ mb: 1 }}>
+                Creator
+              </Typography>
+              <UserChipComponent user={creator} />
+            </Box>
+          </Box>
+      </div>
+      <Paper>
+        <Box>
+          <Box paddingTop={2} paddingBottom={2} paddingLeft={2}>
+            <Grid container direction="row" alignItems="center">
+              <Grid item xs={10}>
+                <Typography variant="h5" align="left">
+                  Personal Information
+                </Typography>
+              </Grid>
+              <Grid item xs={1.9} align="right">
+                <IconButton>
```

Comment:
We should probably make the edit icon blue when the user is actively editing -- and the default gray otherwise. Also if a user clicks the edit button while already editing it should have the same behaviour as clicking the save button.